### PR TITLE
[Tool] Add statistics debug log to find question

### PR DESCRIPTION
**Pull request by Seaven**
Fixes #issue
add debug log to find bug

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by liuyehcf — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java [hunk 1]
```diff
@@ -303,6 +305,22 @@ public List<TStatisticData> executeStatisticDQL(ConnectContext context, String s
...
+            stringBuilder.append("[table: ").append(table.getName()).append(", ");
+            for (Partition partition : table.getPartitions()) {
+                stringBuilder.append(partition.getName()).append(": ");
+                stringBuilder.append(partition.getCommittedVersion()).append(", ");
+                stringBuilder.append(partition.getVisibleVersion()).append(", ");
+                stringBuilder.append(partition.getVisibleVersionTime());
+            }
+            stringBuilder.append("]");
+        }
+        LOG.debug("statistics DQL plan tables: " + stringBuilder);
```
Comment (packy92) [inline fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java:322]: We'd better move the build stringBuilder() process in `if (enableDebug)` block.

Comment (Seaven) [reply fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java]: OK

File: fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java [hunk 2]
```diff
@@ -303,6 +305,25 @@ public List<TStatisticData> executeStatisticDQL(ConnectContext context, String s
     private List<TResultBatch> executeDQL(ConnectContext context, String sql) {
         StatementBase parsedStmt = SqlParser.parseFirstStatement(sql, context.getSessionVariable().getSqlMode());
         ExecPlan execPlan = StatementPlanner.plan(parsedStmt, context, TResultSinkType.STATISTIC);
+
+        List<PhysicalOlapScanOperator> pos = Utils.extractPhysicalOlapScanOperator(execPlan.getPhysicalPlan());
```
Comment (stdpain) [inline fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java:309]: prefer move to debug scope

**Review by packy92 — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 1 / 15 (06.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|1|15|06.67%|[310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323]|

**Pull request merged**